cds-snc/notification-api
# Update Celery config to use UTC time

# Summary | Résumé
This PR updates the Celery and Celery beat configs to enable and use UTC time instead of EST / local time.

# Testing
Do this off main first, or change the celery timezone config back to `America/Toronto` temporarily.
1. Set a distinctive `NOTIFICATION_QUEUE_PREFIX` in your .env file
2. Run the app + Celery + beat
3. Schedule several bulk notifications to yourself in the future, 5 or 10 minutes will do
4. Shut down Celery + beat
5. In staging SQS, check the `<queue-prefix>-bulk-tasks` and `<queue-prefix>-periodic-tasks` queues and verify that the messages are present

*Part 2*
1. Check out this branch or restore the timezone config back to `UTC`
2. Run Celery + beat
3. Receive your scheduled notifications
4. Observe the the queues being drained of tasks/messages, that were scheduled while Celery was configured to use EST time, without issue when configured to use UTC time.
